docs/guide/introduction.md

```diff
@@ -16,7 +16,84 @@ In short, it recognizes pain points on both sides of the CMS experience and reli
 
 ## The module system
 
-Vestibulum id ligula porta felis euismod semper. Maecenas faucibus mollis interdum. Cras mattis consectetur purus sit amet fermentum. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Donec ullamcorper nulla non metus auctor fringilla. Cras mattis consectetur purus sit amet fermentum. Donec ullamcorper nulla non metus auctor fringilla.
+As you learn to use ApostropheCMS, you'll quickly see how all server-side code is organized in a system of "modules." Modules are the building blocks we use to build Apostrophe projects — as well as the CMS itself.
+
+Each module defines a specific set of functionality, from governing the database connection to setting the fields in a blog post. Common Apostrophe project modules will define custom content types, page types, or editable widget types.
```

Comment:
```suggestion
Common Apostrophe project modules will define custom content types, page types, or editable widget types.
```
I removed the first sentence because
1. It seems implicite to me that module define functionality,
2. "governing the database connection" feels abstract and a bit scary.
3. The big value pay off is the second sentence.

Reply:
The point isn't that they defined functionality, but that each one governs a specific set of functionality (they don't share responsibility for things).

Reply:
I'll reverse order to not lead with the complicated one. My point with this is to point out that they really do manage everything, from the simple to complex.